fastly/resource_fastly_service_v1.go

```diff
@@ -1252,6 +1252,8 @@ func resourceServiceV1() *schema.Resource {
 			},
 
 			"httpslogging": httpsloggingSchema,
+
+			"logging_elasticsearch": elasticsearchSchema,
```

Comment:
@ezkl should we stick with the pattern of the `logging` suffix rather than changing to use `logging_` as a prefix, just for consistency with what already exists?

Reply:
We discussed offline. This is the pattern that we would like to use going forward and will be the most prevalent with the staged changes.